index.js

```diff
@@ -118,7 +118,7 @@ Prompt.prototype.render = function(error) {
 
 Prompt.prototype.onSubmit = function(line) {
   var self = this;
-  if (typeof this.opt.validate === 'function' && this.opt.suggestOnly) {
+  if (typeof this.opt.validate === 'function' && (this.opt.suggestOnly || this.opt.alwaysValidate) {
```

Comment:
Aren't you missing a closing `)` here?

Reply:
I am, but I halted work on this because I couldn't get a response from the owner whether this is a feature he wants to add.

Reply:
I'm helping a bit out with maintenance of this library, maybe I can help.

Looking quickly at this feature, it seems like it may be better as the default instead of another option. I'll comment on #59 to this regard.